File: travel_parent/tarvel_web/src/main/webapp/js/controller/CategoryController.js
app.controller("categoryController", function ($scope, categoryService) {


    //根据id查询一条路线
    $scope.findAll=function () {
        categoryService.findAll().success(
            function (response) {
                // alert(response.rows);
                $scope.list = response.rows;
            }
        )
    }
});

File: travel_parent/tarvel_web/src/main/java/com/itheima/travel/controller/CategoryController.java
package com.itheima.travel.controller;

import com.itheima.travel.pojo.Category;
import com.itheima.travel.pojo.PageResult;
import com.itheima.travel.service.CategoryService;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.List;

@RestController
public class CategoryController {

    @Autowired
    private CategoryService categoryService;

    @RequestMapping("/findAll.do")
    public PageResult findAll(){
        List<Category> list =  categoryService.findAll();
        PageResult pr = new PageResult();
        pr.setRows(list);
        return pr;
    }
}
